src/panels/lovelace/components/hui-entity-editor.ts

```diff
@@ -67,15 +67,31 @@ export class HuiEntityEditor extends LitElement {
             ? ""
             : this.entities!.map((entityConf, index) => {
                 return html`
-                  <div class="entity" data-entity-id=${entityConf.entity}>
+                  <div
+                    class="entity"
+                    data-entity-id=${entityConf.entity || entityConf.type}
+                  >
                     <ha-svg-icon .path=${mdiDrag}></ha-svg-icon>
-                    <ha-entity-picker
-                      .hass=${this.hass}
-                      .value=${entityConf.entity}
-                      .index=${index}
-                      @value-changed=${this._valueChanged}
-                      allow-custom-entity
-                    ></ha-entity-picker>
+                    ${entityConf.type
```

Comment:
Thinking we might want to extend this instead, seems very strange to have something specific to only the entities card in this element.

Reply:
Probably not a bad idea

Reply:
We do that and then we can change how we add rows, and give a new prompt instead allowing you choose special row or entity and expand on the entity options

Reply:
Yea that was the plan for this. I have a branch ready to do that and have entity options. But this is the first step